src/spikeinterface/core/core_tools.py

```diff
@@ -239,17 +239,11 @@ def _write_binary_chunk(segment_index, start_frame, end_frame, worker_ctx):
     rec_memmap[start_frame:end_frame, :] = traces
 
 
-def write_binary_recording(
-    recording,
-    file_paths=None,
-    dtype=None,
-    add_file_extension=True,
-    verbose=False,
-    byte_offset=0,
-    auto_cast_uint=True,
-    **job_kwargs,
-):
-    """
+def write_binary_recording(recording, file_paths=None, dtype=None, add_file_extension=True,
+                           verbose=False, byte_offset=0, auto_cast_uint=True, zero_pad_samples=None,
+
+                             **job_kwargs):
+    '''
```

Comment:
```suggestion
    """
```

This is what it is making it fail.